age--1.1.0.sql

```diff
@@ -3793,6 +3793,13 @@ IMMUTABLE
 PARALLEL SAFE
 AS 'MODULE_PATHNAME';
 
+CREATE FUNCTION ag_catalog.age_fact(variadic "any")
```

Comment:
change to CREATE FUNCTION ag_catalog.age_fact(agtype)

Reply:
Changed.